src/entry.c

```diff
@@ -1483,3 +1484,113 @@ void
 unlock_entry( const struct stumpless_entry *entry ) {
   config_unlock_mutex( entry->mutex );
 }
+
+
+/*
+ * formating entry_to_string values
```

Comment:
Complete the documentation of this function as well, to make it easier to use elsewhere in the future.